### DISCUSSION TITLE
Volume control does not work through the HDMI port of M1 macs

### DISCUSSION THREAD

**Discussion by ghost** [MonitorControl/MonitorControl#1113] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

The volume slider doesn't work at all on my screen. Only physical keys can change volume.

### Expected behavior

Using the slider in MonitorControl should change volume.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.4
- Mac model: M1 mini
- MonitorControl version: latest
- Monitor(s): Just one, C49J89x
- Monitor Cable(s)/Connection(s): HDMI
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
If you connect the display via the HDMI port of the M1 Mini, DDC control won't work as this port does not relay DDC commands. You need to use USB-C. If the display firmware supports it, you'll be able to control the volume that way.

**Reply by ghost** [reply]
> If you connect the display via the HDMI port of the M1 Mini, DDC control won't work as this port does not relay DDC commands. You need to use USB-C. If the display firmware supports it, you'll be able to control the volume that way.

I tried what you suggested. It didn't work.

Moreover, moving the volume slider completely crashed the display, i.e., constant flickering, due to which I had to hard-shut down the Mac.

The same happened when I tried to "mute" by bringing the slider all the way to the left.

It's buggy on my system, I had to uninstall. Sorry, no further feedback will be provided from my end.

**Reply by waydabber** [reply]
All right. The display most probably has a faulty firmware and does not handle DDC commands properly.

**Comment by ssbarnea** [comment]
While the readme specifies that for M1 MacBook (MacBookPro18,2), DDC should work for USB-C to HDMI adapters, I can report that at least for the one that I have it does not have any effect. The device is not visible inside the USB or Thunderbolt/USB4 tabs but I can see it inside Audio one, listed as 
```
marantz-AVR:

  Default Output Device:	Yes
  Manufacturer:	MJI
  Output Channels:	8
  Current SampleRate:	48000
  Transport:	HDMI
  Output Source:	Default
```

Maybe someone can report if maybe another brand that works, mine seems to be a "MJI".
